### DISCUSSION TITLE
Air Gapped Installed

### DISCUSSION THREAD

**Discussion by bgoralczyk** [jhuckaby/Cronicle#816] [Q&A]
Is there any documentation regarding installing this application on an air gapped server?  I can installed node and npm without issue.  I can get the source down to my server without issue, but the npm installed command runs but doesn't seem to ever end.

**Comment by jhuckaby** [comment]
The best way is to install it on a normal server first, then zip up the entire `/opt/cronicle` directory, and copy that over to your air-gapped server, then decompress it and run it.
